app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,58 @@
+
+
+<section id="index_container">
+  <h2 id="to_do"> # TODO</h2>
+  <ul class="to_do_list">
+    <% counter = 0 %>
+    <% @tasks.each do |task| %>
+    <% counter += 1 %>
+    <li>
+      <%= counter %>)
```

Comment:
it sounds like you had a specific way of formatting these numbers anyway, but you should/can consider that you could have made this an ordered list instead of unordered (line 5, `<ol>` instead of `<ul>`), and then gotten the numbering for free without needing to create a counter